client/landing/stepper/declarative-flow/link-in-bio.ts

```diff
@@ -20,7 +26,16 @@ export const linkInBio: Flow = {
 			recordFullStoryEvent( 'calypso_signup_start_link_in_bio', { flow: this.name } );
 		}, [] );
 
-		return [ 'intro', 'linkInBioSetup', 'patterns', 'processing', 'launchpad' ] as StepPath[];
+		return [
+			'intro',
+			'linkInBioSetup',
+			'domains',
+			'patterns',
+			'siteCreationStep',
+			'processing',
+			'launchpad',
+			'plans',
```

Comment:
It's not a must, but having the steps in order is better.
```suggestion
			'intro',
			'linkInBioSetup',
			'domains',
			'plans',
			'patterns',
			'siteCreationStep',
			'processing',
			'launchpad',
			
```